src/rencache.c

```diff
@@ -159,7 +160,7 @@ void rencache_invalidate(void) {
 }
 
 
-void rencache_begin_frame(lua_State *L) {
+void rencache_begin_frame(UNUSED lua_State *L) {
```

Comment:
Ditto above.

Reply:
removing it moved the UNUSED from this to `f_begin_frame`

Reply:
Wouldn't those `lua_State`s be useful in case we add support for multiple windows?

Reply:
Sure; but we can always add them back in when and if that happens. As it stands, if it's causing a warning, let's just nix it.

Reply:
It might be nice to keep so that the signature of all rencache functions stays similar.
We cannot remove `lua_State` from the `f_{begin,end}_frame` functions so it might be worth to keep this